db/agendadb/db.go

```diff
@@ -220,13 +220,20 @@ func GetAllAgendas() (agendas []*AgendaTagged, err error) {
 	adb, err := Open(dbName)
```

Comment:
I believe err here shadows the err declared in the function signature

Reply:
Ok. Noted